src/pages/terms/[termCode].astro

```diff
@@ -0,0 +1,16 @@
+---
+import json from "../../data/terms.json";
+export function getStaticPaths() {
+  const terms = json;
+  return terms.map(({ termCode, name, alternateName }) => {
+    return {
+      params: { termCode },
```

Comment:
I'm not sure if termCode is what we want to use here. Didn't get a good grasp of what it meant from schema's documentation other than some sort of unique ID.

Not going against this, just raising it up. The param should obviously be unique.